<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exp4Window</class>
 <widget class="QWidget" name="Exp4Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>开始实验</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>451</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
